client/components/tooltip/docs/example.jsx

```diff
@@ -10,7 +10,7 @@ class Tooltip extends PureComponent {
 		this.open = this.open.bind( this );
 		this.close = this.close.bind( this );
 		this.changePosition = this.changePosition.bind( this );
-
+		this.tooltip = createRef();
```

Comment:
We usually name these `tooltipRef`, kind of a Hungarian convention to encode the type in the name.